etc/profile-a-l/lobster.profile

```diff
@@ -8,8 +8,14 @@ include lobster.local
 # added by included profile
 #include globals.local
 
+ignore private-dev
+
+noblacklist ${HOME}/.cache/ueberzugpp
 noblacklist ${HOME}/.config/lobster
+noblacklist ${HOME}/.config/ueberzugpp
+noblacklist ${HOME}/.local/share/applications/lobster
 noblacklist ${HOME}/.local/share/lobster
+noblacklist ${PATH}/openssl
```

Comment:
Is this line actually needed?

It is only blacklisted in disable-devel.inc, which is not included in here.

Other than that, LGTM.

Reply:
> It is only blacklisted in disable-devel.inc, which is not included in here.

It's included in mpv.profile, so we do need it here.